sdk/pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>com.horizen</groupId>
             <artifactId>zendoo-sc-cryptolib</artifactId>
-            <version>0.2.1-SNAPSHOT</version>
+            <version>0.2-SNAPSHOT</version>
```

Comment:
is it correct? Why not 0.2.6 generated yesterday?

Reply:
we need to define it, current version in development branch in zensoo-sc-cryptolib is 0.2-SNAPSHOT